server/webapp/WEB-INF/rails.new/app/views/admin/tasks/fetch/new.html.erb

```diff
@@ -1,9 +1,23 @@
-<%= render :partial => "admin/tasks/fetch/form.html", :locals => {:scope => scope} %>
+<%= render :partial => "admin/tasks/fetch/form.html", :locals => {:scope => {:task => scope[:task],
```

Comment:
new.html.erb and edit.html.erb seem to have the same content except for routeFetch and routeFetchPluggable. Consider moving to a partial and providing the values for routeFetch and routeFetchPluggable?

Reply:
I rather keep those `erbs`. I am fine with those duplicated lines.